src/Equinor.ProCoSys.IPO.Query/GetInvitationById/GetInvitationByIdQueryHandler.cs

```diff
@@ -280,37 +276,8 @@ private static InvitedPersonDto ConvertToInvitedPersonDto(Participant participan
         private static IEnumerable<InvitedPersonDto> ConvertToInvitedPersonDto(IEnumerable<Participant> personsInFunctionalRole) 
             => personsInFunctionalRole.Select(ConvertToInvitedPersonDto).ToList();
 
-        private bool IsSigningParticipant(Participant participant)
```

Comment:
Resharper complains on line 312:
![image](https://user-images.githubusercontent.com/19545528/111277018-45118a00-8638-11eb-97c5-8ee18ad8b989.png)
I'm not sure why, but can you have a look?

Reply:
These are known to be not null. However, at one point we did have an issue with the fusion response where it seemed like Person was in fact null. The reason why I keep these '?' is just in case we need to check the log. This is only temporary code, and there is a PBI to remove the logging. Would like to keep it just in case